src/app/models/tag.py

```diff
@@ -1,6 +1,7 @@
 from django.db import models
 
 class Tag(models.Model):
+    id = models
```

Comment:
I feel like this would error out. `id` is included by default so we can remove that attribute.

Reply:
Ok, I can remove.